source/docs/api-reference/device-specific/talonfx/closed-loop-requests.rst

```diff
@@ -7,9 +7,12 @@ Since closed-loop control changes based on the dynamics of the system (velocity,
 
 Manual tuning typically follows this process:
 
-1. Set :math:`K_p`, :math:`K_i` and :math:`K_d` to zero.
-2. Increase :math:`K_p` until the output starts to oscillate around the setpoint.
-3. Increase :math:`K_d` as much as possible without introducing jittering to the response.
+1. Set all gains to zero.
+2. Determine :math:`K_g` if using an arm or elevator.
+3. Increase :math:`K_s` until just before the motor moves.
```

Comment:
This won't work if kS mode is velocity sign and the user has a 0 velocity setpoint